### DISCUSSION TITLE
Error! Class in action not comply with psr-4 autoloading standard in Tomado

### DISCUSSION THREAD

**Discussion by Queopius** [tomatophp/tomato#26] [General]
![class in action not comply with psr-4 autoloading standard in Tomado 2024-02-20 19-00-33](https://github.com/tomatophp/tomato/assets/10995315/6cd55196-3678-4ac5-be04-504fa9a7cf87)

**Comment by Queopius** [comment]
Ao clonar o repositório Tomato em local me sai esse erro. 
Talvez fosse interessante corrigir diretamente desde o repositório.

**Comment by fadymondy** [comment]
it's not an error it's a warning one because of PSR-4 it can be fixed by just changing the name of actions.
